timescaledb/tutorials/nyc-taxi-cab/query-nyc.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Query time-series data tutorial - query the data
+excerpt: Query time-series data
+keywords: [tutorials, query]
+tags: [tutorials, beginner]
+---
+
+# Query the data
+
+When you have your dataset loaded, you can start constructing some queries to
+discover what your data tells you. In this section, you learn how to write
+queries that answer these questions:
+
+*   [How many rides take place each day?](#how-many-rides-take-place-every-day)
+*   [What is the average fare amount?](#what-is-the-average-fare-amount)
+*   [How many rides of each rate type were taken?](#how-many-rides-of-each-rate-type-were-taken)
+*   [What kind of trips are going to and from airports?](#what-kind-of-trips-are-going-to-and-from-airports)
+*   [How many rides took place on New Year's Day 2016](#how-many-rides-took-place-on-new-years-day-2016)?
+
+## How many rides take place every day?
+
+This dataset contains ride data for January 2016. To find out how many rides
+took place each day, you can use a `SELECT` statement. In this case, you want to
+count the total number of rides each day, and show them in a list by date.
+
+<procedure>
+
+### Finding how many rides take place every day
+
+1.  Connect to the Timescale Cloud database that contains the NYC taxi dataset.
+1.  At the psql prompt, use this query to select all rides taken in the first
+    week of January 2016, and return a count of rides for each day:
+
+    ```sql
+    SELECT date_trunc('day', pickup_datetime) as day,
```

Comment:
I'm pretty happy with where time_bucket is at the moment, because the queries build nicely towards it. I think the whole tutorial would need to be rewritten to make it work earlier (which is something we might decide we want to do, but I'm not really prepared to do it now).